Hummingbird.docc/Tutorials/FluentUniverse/Fluent.tutorial

```diff
@@ -0,0 +1,11 @@
+@Tutorials(name: "Galaxy Backend") {
+    @Intro(title: "Store Data with Fluent") {
+        A tutorial that shows you how to set up Hummingbird 2 with Fluent to store, access, edit and delete your Galaxy.
```

Comment:
Tutorial doesn't show you how to edit and delete galaxies

Reply:
I should change that, I wanted to add that here, but ended up deciding to keep it a bit leaner for now. Do you think edit/delete should be part of this 1st part, or is it fine for part 2?

Reply:
You could add a second part, but edit would need some attributes to edit. Maybe you could add another migration that added an attribute to each galaxy.